Makefile

```diff
@@ -260,13 +260,13 @@ ubuntu:
 build-ubuntu:
 	@echo "===> Building Antrea bins and antrea/antrea-ubuntu Docker image <==="
 ifneq ($(DOCKER_REGISTRY),"")
-	docker build -t antrea/antrea-ubuntu:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.ubuntu .
+	sudo docker build -t antrea/antrea-ubuntu:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.ubuntu .
```

Comment:
I don't think it is a good idea to add "sudo" in `Makefile`. Instead I suggest putting it in scripts.

Reply:
The issue here lies in that the sudoer for the jenkins user does not have full root privileges, it only has privilege for selected commands for security reasons. if the `make` command is given privilege then the user will basically have full root privilege, the easiest way to fix this is to add the sudo command inside the `Makefile` since `docker` is added to the allowed command, if you have other suggestions i am more than happy to hear them.

Reply:
We are not going to add `sudo` in the Makefile, especially given that this target is commonly used by all developers.

If you do not want to add the jenkins user to the docker group for security reasons, I suggest:
- trying to use the new docker rootless mode (https://docs.docker.com/engine/security/rootless/)
- duplicating these specific build commands in a script for your usage, as @lzhecheng suggested

Reply:
Thanks for the suggestions.

Removed the sudo from the make file and added the docker build to the scripts. Please note that if the Dockerfile location is changed in the future it will break the scripts, so if they are changed, they must be changed in the scripts also.

Thanks